README.md

```diff
@@ -68,6 +68,26 @@ If you would like the plugin to attempt to annotate the first HTML element creat
       ["@fullstory/babel-plugin-annotate-react", { "annotate-fragments": true }]
     ]
 
+If you would like the plugin to skip the annotation for certain components, use the `ignore-components` option:
+
+```javascript
+  plugins: [
+      [
+        '../..',
+        { 
+          "ignore-components":[
+            // each item must be a string array containing three items: file name, component name, element name 
+            // corresponding to the values for data-source-file, data-component, data-element
+            // use wild card (*) to match anything
+            ["myBoxComponent.jsx","MyBox","Box"],
```

Comment:
Nice. This is exactly what I was thinking we would need to provide so that we can handle scenarios where only the component name is know and scenarios where there is a collision and more info needs to be added.